config/examples/3DFabXYZ/Migbot/Configuration_adv.h

```diff
@@ -1220,6 +1245,9 @@
   // Use a numeric passcode for "Screen lock" keypad.
   // (recommended for smaller displays)
   //#define TOUCH_UI_PASSCODE
+
+  // Enable this for additional debugging information
```

Comment:
The configurations will form the basis for the documentation on the website. So we can leave off phrases like "Enable this for..." and skip straight to stating the effect the setting will have. Prefer small words. For example:

"Output extra debug info for Touch UI events"